test/unit/core/diffusion_test.cc

```diff
@@ -1045,7 +1045,7 @@ TEST(DiffusionTest, EulerNeumannZeroBoundaries) {
     rm->AddContinuum(dgrid);
 
     // Simulate diffusion / exponential decay for `tot` timesteps
-    int tot = 10000;
+    int tot = 1000;
```

Comment:
Have you checked if the concentration still reaches the boundary @ryan-bournes ? This test is supposed to verify that nothing escapes with homogeneous Neumann conditions, the test is only meaningful if the concentration reaches the boundaries. Possibly a good idea to lower `int tot` but add a source close to each of the six boundaries.